src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/FlowMapper.java

```diff
@@ -79,6 +79,7 @@
     @Mapping(target = "forwardLatency", ignore = true)
     @Mapping(target = "reverseLatency", ignore = true)
     @Mapping(target = "latencyLastModifiedTime", ignore = true)
+    @Mapping(target = "yFlowId", source = "YFlowId")
```

Comment:
Looks excess, mapstruct can find this relation by himself.

Reply:
No. In our Mapstruct configuration, the AccessorNamingStrategy is case sensitive. And here, the source is a getter method "getYFlowId", so the mapper sees it as "YFlowId". The target is a builder method "yFlowId". They are different from the mapper standpoint.